releases/release-7.2.0.md

```diff
@@ -0,0 +1,341 @@
+---
+title: TiDB 7.2.0 Release Notes
+summary: Learn about the new features, compatibility changes, improvements, and bug fixes in TiDB 7.2.0.
+---
+
+# TiDB 7.2.0 Release Notes
+
+Release date: xx xx, 2023
+
+TiDB version: 7.2.0
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.2/quick-start-with-tidb) | [Installation packages](https://www.pingcap.com/download/?version=v7.2.0#version-list)
+
+7.2.0 introduces the following key features and improvements:
+
+<table>
+<thead>
+  <tr>
+    <th>Category</th>
+    <th>Feature</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td rowspan="2">Scalability and Performance</td>
+    <td>Runaway queries managed by Resource Groups (experimental)</td>
+    <td>Query timeouts exist on a per-TiKV node basis but now Resoure Groups can manage queries by global parallel execution time and either deprioritize or kill them. Allowing operators to configure this timeout and what do with the queries at a Resource Group level, gives them much more control over the impact unexected large queries may have on the cluster.
```

Comment:
@Connor1996 PTAL for accuracy.

Reply:
Query timeout exists before as MaxExecutionTime. So don't need to compare it with `Query timeouts exist on a per-TiKV node`